policyengine_uk/variables/gov/dfe/study_childcare_entitlement/study_childcare_entitlement_eligible.py

```diff
@@ -0,0 +1,43 @@
+from policyengine_uk.model_api import *
+
+
+class study_childcare_entitlement_eligible(Variable):
+    value_type = bool
+    entity = Person
+    label = "eligible for study childcare entitlement through Care to Learn childcare support"
+    definition_period = YEAR
+
+    def formula(person, period, parameters):
+        # Link for instruction: https://www.gov.uk/care-to-learn/eligibility
+
+        # Only parents can be eligible, not children
+        is_child = person("is_child", period)
+
+        # Check basic eligibility conditions
+        has_children = person.benunit.any(person("is_child", period))
+        p = parameters(
+            period
+        ).gov.dfe.study_childcare_entitlement.care_to_learn
+        age_eligible = person("age", period) < p.age_limit
+
+        current_ed = person("current_education", period)
+        education_types = current_ed.possible_values
+
+        # Only exclude higher education/tertiary
+        not_higher_education = current_ed != education_types.TERTIARY
+
+        not_apprentice = ~person("is_apprentice", period)
```

Comment:
then use `~is_apprentice` for consistency with `~is_child`
```suggestion
        is_apprentice = person("is_apprentice", period)
```

Reply:
why did you resolve this without doing it or explaining why you didn't

Reply:
Previously we had ```is_child```. For being consistent with ```~is_child``` in the conditions, you suggest to define ```is_apprentice```. 

As now we have ```is_parent```, if we want to be consistent with this variable, it's better to have ```not_apprentice``` (I assume you meant we don't want to use ```~``` in ```return```.